Sources/Amplitude/Amplitude.m

```diff
@@ -970,10 +928,10 @@ - (void)makeEventUploadPostRequest:(NSString *)url events:(NSString *)events num
                 // If blocked by one massive event, drop it
                 if (numEvents == 1) {
                     if (maxEventId >= 0) {
-                        (void) [self.dbHelper removeEvent:maxEventId];
+                        self->_eventsBuffer = [[NSMutableArray alloc] init];
```

Comment:
If it falls into this case, do we need to anything about the file? I guess the massive event in this case is already saved on the file?

Reply:
Also fixed this.